src/main/resources/templates/kotlin/test/projectTemplate/pom.xml

```
@@ -102,6 +102,44 @@
                     <target>17</target>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <version>3.0.0</version>
+                <executions>
+                    <execution>
+                        <id>enforce-no-student-code-in-trusted-packages</id>
+                        <phase>process-classes</phase>
+                        <goals>
+                            <goal>enforce</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <rules>
+                        <requireFilesDontExist>
+                            <files>
+                                <file>${project.build.outputDirectory}/ch/qos/logback/</file>
+                                <file>${project.build.outputDirectory}/com/intellij/</file>
+                                <file>${project.build.outputDirectory}/com/sun/</file>
+                                <file>${project.build.outputDirectory}/de/tum/in/test/api/</file>
+                                <file>${project.build.outputDirectory}/java/</file>
+                                <file>${project.build.outputDirectory}/javax/</file>
+                                <file>${project.build.outputDirectory}/jdk/</file>
```

```suggestion
                                <file>${project.build.outputDirectory}/jdk/</file>
                                <file>${project.build.outputDirectory}/kotlin/</file>
```
This fixes the failing server tests.

---

Thanks for pointing that out. 👍